Packs/PAN-OS/ReleaseNotes/2_1_10.md

```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+
+##### PAN-OS - Block Destination Service
+
+Added the `SecondaryDeviceGroup` input in case we want to add the block rule to the "Shared" location put still push to applicable device groups.
```

Comment:
Added the `SecondaryDeviceGroup` input in case we want to add the block rule to the "Shared" location put still push to applicable device groups.

Did you mean '...and still push to...' ?

Reply:
yes, thank you, will change

Reply:
changed in [update RN](https://github.com/demisto/content/pull/30156/commits/780f49356bc1f817723527009d55d2242f01c53c)

Reply:
Hey, thank you, please add the hyphen (-) to the RN, you accidentally deleted it and the format is wrong now.